app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class SessionsController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
```

Comment:
That being said, line 9 could get abstracted to a method like so:

def log_in
  session[:user_id] = @user.id
end